### PR TITLE
Fix spec and man

### PR DESCRIPTION
**Pull request by runcom**
- install man8 (you can't do `man dockerd` nor `man docker daemon` otherwise)
- rename `man/config-json.5.md` to be `docker-` prefixed - no sense in ppl doing `man config-json`, instead they probably do `man docker config-json`

### CODE REVIEW

**Comment by thaJeztah** [comment]
sgtm, was looking for something similar in the debs configuration, but couldn't find it

ping @tianon do you know if changes need to be made for the debs?

**Comment by runcom** [comment]
ping @thaJeztah @tianon

**Comment by thaJeztah** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM 🐹

File: hack/make/.build-rpm/docker-engine.spec [hunk 1]
```diff
@@ -165,6 +165,8 @@ install -d %{buildroot}%{_mandir}/man1
 install -p -m 644 man/man1/*.1 $RPM_BUILD_ROOT/%{_mandir}/man1
 install -d %{buildroot}%{_mandir}/man5
 install -p -m 644 man/man5/*.5 $RPM_BUILD_ROOT/%{_mandir}/man5
+install -d %{buildroot}%{_mandir}/man8
+install -p -m 644 man/man5/*.8 $RPM_BUILD_ROOT/%{_mandir}/man8
```
Comment (thaJeztah) [inline hack/make/.build-rpm/docker-engine.spec]: this should've been `man/man8/*.8`. opening a PR to fix

**Pull request merged**